### PR TITLE
Popuptemplate

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: templates/pop/editors/editor.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+			background-repeat: no-repeat;
+			-moz-background-size: 12px 12px;
+			background-size: 12px 12px;
+			background-position: 90%;
+			cursor: pointer;
+			padding-left: 5px;
+			border-radius: 7px;
+			background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIgd2lkdGg9IjEwMCIgaGVpZ2h0PSIxMDAiPjxwb2x5Z29uIHBvaW50cz0iMCwwIDUwLDg2LjYgMTAwLDAiIGZpbGw9IiMzMzMiLz48L3N2Zz4%3D');
+			/*
+			<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="100" height="100"><polygon points="0,0 50,86.6 100,0" fill="#333"/></svg>
```
Comment (jbuck) [inline templates/pop/editors/editor.css:32]: wat

Comment (secretrobotron) [inline templates/pop/editors/editor.css:32]: right.

**Comment by benrito** [comment]
this. is. SLICK!

**Comment by dseif** [comment]
This just landed in Popcorn interim, closing.

**Pull request closed without merging**